src/pincode/PincodeSet.test.tsx

```diff
@@ -166,7 +166,7 @@ describe('Pincode', () => {
     })
 
     expect(updatePin).toHaveBeenCalledWith(mockAccount.toLowerCase(), oldPin, mockPin)
-    expect(navigate).toBeCalledWith(Screens.Settings)
+    expect(navigate).toBeCalledWith(Screens.SettingsDrawer)
```

Comment:
better to switch any test that checks settings navigation to run against the `true` variant (or test both scenarios). Might also be worth wrapping navigating to settings into a helper function given my comments above. I don't think it will be as simple as navigating one or the other screen